reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md

```diff
@@ -23,66 +23,158 @@ other programs. For example, the `WINDIR` environment variable contains the
+  multiple instances of PowerShell installed, the filename will include a
+  hexadecimal suffix so that there is a a unique filename per installation.
+
+  To change the default location of the cache, set the environment variable
+  before starting PowerShell. Changes to this environment variable only affect
+  child processes. The value should name a full path (including filename)
+  that PowerShell has permission to create and write files.
+
+  To disable the file cache, set this value to an invalid location, for
+  example:
+
+  ```powershell
+  # `NUL` here is a special device on Windows that cannot be written to
+  $env:PSModuleAnalysisCachePath = 'NUL'
+  ```
+
+  This sets the path to the **NUL** device. PowerShell can't write to the
+  path but no error is returned. You can see the errors reported using a
+  tracer:
+
+  ```powershell
+  Trace-Command -PSHost -Name Modules -Expression { Import-Module Microsoft.PowerShell.Management -Force }
+  ```
+
+- PSDisableModuleAnalysisCacheCleanup
+
+  When writing out the module analysis cache, PowerShell checks for modules
+  that no longer exist to avoid an unnecessarily large cache. Sometimes these
+  checks are not desirable, in which case you can turn them off by setting this
+  environment variable value to `1`.
+
+  Setting this environment variable takes effect immediately in the current
+  process.
+
+- PSModulePath
+
+  The `$env:PSModulePath` environment variable contains a list of folder
+  locations that are searched to find modules and resources.
+
+  By default, the effective locations assigned to `$env:PSModulePath` are:
+
+  - System-wide locations: These folders contain modules that ship with
+    PowerShell. The modules are store in the `$PSHOME\Modules` location. Also,
+    This is the location where the Windows management modules are installed.
+
+  - User-installed modules: These are modules installed by the user.
+    `Install-Module` has a **Scope** parameter that allows you to specify
+    whether the module is installed for the current user or for all users. For
+    more information, see [Install-Module](../../PowerShellGet/Install-Module.md).
+
+    - On Windows, the location of the user-specific **CurrentUser** scope is
+      the `$HOME\Documents\PowerShell\Modules` folder. The location of the
+      **AllUsers** scope is `$env:ProgramFiles\PowerShell\Modules`.
+
+  In addition, setup programs that install modules in other directories, such
+  as the Program Files directory, can append their locations to the value of
+  `$env:PSModulePath`.
+
+  For more information, see [about_PSModulePath](about_PSModulePath.md).
+
+## Managing environment variables
+
+Environment variables can be access and changed using several different methods
+in PowerShell.
+
+- You can use the Environment provider drive.
+- You can use the Item cmdlets to get or set values.
+- You can use the .NET **System.Environment** class.
+- On Windows, you can also use the System Control Panel.
+
```

Comment:
Env var prefix like `$env:psmodulepath` is technically different than `env:\psmodulepath` although the result is the same, but should be called out as a different method